src/common/MessageHandler.h

```diff
@@ -75,6 +77,14 @@ namespace SDDM {
                 file.open(QFile::Truncate | QFile::WriteOnly);
         }
 
+        // If we can't open the file, create it in a writable location
```

Comment:
For the non-QT experts, can you provide an inline example where this "writeable location" might be?

Reply:
You can test it yourself:
$ qtpaths --writable-path AppDataLocation
/home/apol/.local/share/<APPNAME>

Reply:
Neat, can you add an inline example?

Reply:
Added examples on the comment.